types/index.test-d.ts

```diff
@@ -0,0 +1,282 @@
+import OktaSignIn, { OktaSIW } from '@okta/okta-signin-widget';
+import { OktaAuth, Tokens } from '@okta/okta-auth-js';
+import { expectType, expectError, expectAssignable } from 'tsd';
+/* eslint-disable camelcase */
```

Comment:
this can be fixed
- change variable name
- `schema.profileSchema.properties['country_code'] = countryCode;`